Add shot limit on HUD

The-Noah opened the pull request:
Add the number of shots left for the current flag on the HUD. Remove the messages saying the number of shots left.

The-Noah committed:
Add shot limit on HUD

Add the number of shots left for the current flag on the HUD. Remove the messages saying the number of shots left.

The-Noah committed:
Update shot limit label

Change the shot limit label location to be on top of the shot indicators instead of to the right of them. Also make them slightly transparent. Fix problem where not showing the currect number when observing.

allejo commented:
Since this affects the client visually, please share screenshots of what your changes do. This way we get an idea of how it'll look and be able to give you feedback all without having to download your patch and test it ourselves.

The-Noah-BZ commented:
![bzfi0019](https://user-images.githubusercontent.com/35819660/43337592-017f3f8e-919a-11e8-9a72-ea3b9e902f26.png)

JeffM2501 commented:
It needs some kind of label or icon next to the number showing that it's a limit, not just a random floating number.

The-Noah committed:
Add 'shots left' label

When the player grabs a flag with a limited number of shots, the label will fade in for 1 second, show for 3, then fade out for 1 second.

The-Noah committed:
Merge branch 'master' into shot-limit-hud

allejo commented:
I'm personally hesitant of the location of this label. Thoughts on placement? I'd favor a place where other flag information is already stored, e.g.

- Bottom of the mouse box, above where flag help messages would appear
- Upper right next to the flag name

The-Noah-BZ commented:
Hmm. I could add a simple option to switch it's location between where you said, that way if the player doesn't like it in one spot he/she could move it to the other. What do you think?

JeffM2501 commented:
BZFlag does not need more options.
Show us screenshots or mockups with both locations.
I prefer lower near the shot reloads, since that's already shot related.

But there may be more room near the flag

The-Noah committed:
Merge pull request #6 from BZFlag-Dev/master

Master

The-Noah committed:
Merge pull request #9 from BZFlag-Dev/master

Master

allejo commented:
Since this PR has grown stale and it's something we're going to need to discuss further with some more mockups, I'm placing this one on hold.

TheGiraffe3 commented:
So what's the status on this?
